controllers/cleanupoperator_controller.go

```diff
@@ -174,8 +193,40 @@ func (r *CleanUpOperatorReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 				log.Info("NetApp Trident Template Cleaned Successfully!!!")
 
 			case template == "local-volume":
-				log.Info("Local Volume cleanup not yet supported.")
-				return ctrl.Result{}, nil
+				fmt.Println("Local Volume")
+
+				err = r.localVolumeCleanUp(ctx, namespace)
+				if err != nil {
+					// Failed to perform CleanUp
+					return ctrl.Result{}, err
+				}
+				err = r.removeLocalVolmeCRDs(ctx)
+				if err != nil {
+					// Failed to remove CRDs
+					return ctrl.Result{}, err
+				}
+
+				localVolumeOpFound := true
+				localVolumeOperator := &appsv1.Deployment{}
+				err = r.Get(ctx, types.NamespacedName{Name: localVolumeOperatorName, Namespace: namespace}, localVolumeOperator)
+				if err != nil {
+					if errors.IsNotFound(err) {
+						log.Info("Local Volume Operator not found. Ignoring...", "name", localVolumeOperatorName)
+						localVolumeOpFound = false
+					}
+					log.Error(err, "Failed to get Local Volume Operator", "name", localVolumeOperatorName)
```

Comment:
Put this in `else` part.
Please update same for `netapp`. This is a bug in code.

Reply:
else {
  log.Error(err, "Failed to get Local Volume Operator", "name", localVolumeOperatorName)
  return ctrl.Result{}, err
}

Reply:
Done